tests/integration/ContentTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Transfer\EzPlatform\Tests\Integration;
+
+use Transfer\Adapter\Transaction\Request;
+use Transfer\EzPlatform\Adapter\EzPlatformAdapter;
+use Transfer\EzPlatform\Data\ContentObject;
+use Transfer\EzPlatform\Tests\EzPlatformTestCase;
+
+class ContentTest extends EzPlatformTestCase
+{
+    /**
+     * @var EzPlatformAdapter
+     */
+    protected $adapter;
+
+    public function setUp()
+    {
+        $this->adapter = new EzPlatformAdapter(array(
+            'repository' => static::$repository,
+        ));
+    }
+
+    /**
+     * Tests content creation
+     */
+    public function testCreateContent()
+    {
+        $contentObject = new ContentObject(
+            array(
+                'title' => 'Test title',
+            ),
+            array(
+                'language' => 'eng-GB',
+                'content_type_identifier' => '_test_article',
+                'remote_id' => 'test_article_1'
+            )
+        );
+
+        $this->adapter->send(new Request(array(
+            $contentObject
+        )));
+
+        $content = static::$repository->getContentService()->loadContentByRemoteId('test_article_1');
+
+        $this->assertInstanceOf('\eZ\Publish\API\Repository\Values\Content\Content', $content);
+        $this->assertEquals('Test title', $content->contentInfo->name);
+        $this->assertEquals('Test title', $content->fields['title']['eng-GB']->text);
+        $this->assertEquals('eng-GB', $content->contentInfo->mainLanguageCode);
+        $this->assertEquals(36, $content->contentInfo->contentTypeId);
+    }
+
+    /**
+     * Tests content update
+     */
+    public function testUpdateContent()
+    {
+        $contentObject = new ContentObject(
+            array(
+                'title' => 'Test updated title',
+            ),
+            array(
+                'language' => 'eng-GB',
+                'content_type_identifier' => '_test_article',
+                'remote_id' => 'test_article_1'
+            )
+        );
+
+        $this->adapter->send(new Request(array(
+            $contentObject
+        )));
+
+        $content = static::$repository->getContentService()->loadContentByRemoteId('test_article_1');
+
+        $this->assertInstanceOf('\eZ\Publish\API\Repository\Values\Content\Content', $content);
+        //$this->assertEquals('Test updated title', $content->contentInfo->name);
```

Comment:
You were right about the cache. Seems that the persistant cache is failing this one. Still since you passed a field update, you should test towards the field as I stated in the comment above. Its [this](https://github.com/transfer-framework/ezplatform/blob/b46adfdc99a4b75f01df99aedc5b64a258c8d79f/src/Transfer/EzPlatform/Repository/Manager/ContentManager.php#L181) line that creates the cache, which only seems to be there for the logging. The contentinfo fetch could be placed in the update-function instead (if null).

Also, I think we should use Content\Content->getFields() and getField methods.